### PR TITLE
Add Windows Compability!

### LINKED ISSUE #21: Windows support?
Hey @hemanth! I heard that this doesn't work on Windows. What needs to happen to get this to work on Windows?

### PR DESCRIPTION
**Pull request by bhaveshgohel**
Added some minor changes to @kentcdodds 's PR https://github.com/hemanth/node-nightly/pull/22 to work with Windows.

/cc: @hemanth @tarungarg546 

Closes #21

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,11 +20,11 @@ module.exports = {
 		return nodeNightlyVer.then(latest => {
 			const conf = new Configstore(pkg.name);
 			conf.set('version', latest);
-			const os = process.platform;
+			const os = process.platform  === 'win32' ? 'win' : process.platform;
 			const arch = process.arch;
 			const type = 'nightly';
 			osArchString = `${latest}-${os}-${arch}`;
-			const url = `https://nodejs.org/download/${type}/${latest}/node-${osArchString}.tar.gz`;
+			const url = `https://nodejs.org/download/${type}/${latest}/node-${osArchString}.zip`;
```
Comment (kentcdodds) [inline index.js]: I think that the extension needs to be dependant on the platform. I'm pretty sure that `zip` should only be used for windows.

But this is really close!

Comment (bhaveshgohel) [inline index.js]: yeah, but as of right now node.org only has .zip files uploaded so i added .zip, i'm sure we can use .tar.gz if they had uploaded binaries in that extension for windows. :+1:

Comment (bhaveshgohel) [inline index.js]: Oh Got it! :+1: 
You mean to say like '.zip' for 'win' and '.tar.gz' for other? [ can be done using if/else]

**Comment by kentcdodds** [comment]
I'm fairly certain this will work! Would be awesome for my workshop attendees to be able to use this for the workshop today!!! <3 We'll need it in a couple of hours. I understand if you don't have time though :) Thanks!

**Comment by bhaveshgohel** [comment]
@kentcdodds can you please help here: https://github.com/hemanth/node-nightly/pull/23#issuecomment-238211670

**Comment by kentcdodds** [comment]
Note, these days GitHub has a squash and merge option, so @hemanth can do that and you shouldn't need to clean up your commit history.

**Comment by bhaveshgohel** [comment]
@kentcdodds i just see those videos and do as it say and also squash my commit using this [Reference](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

What is left from my side? :confused:

**Comment by kentcdodds** [comment]
Well, you still have a bunch of commits in this pull request, so it looks like you haven't actually squashed your commits. But like I said, @hemanth can squash it for you with the GitHub UI so you should be fine. :-)

**Comment by bhaveshgohel** [comment]
@kentcdodds Thank you for support & help. :smiley:

**Comment by hemanth** [comment]
Yup, squashing is no more a problem.

**Comment by hemanth** [comment]
#24 looks good to me, let us close #23 and merge this?

**Comment by kentcdodds** [comment]
FYI @bhaveshgohel responded with 👍 to your comment @hemanth :)

**Comment by bhaveshgohel** [comment]
@hemanth You can merge this PR. :+1:

**Comment by hemanth** [comment]
em4i0bDs9Hm2Q

This deserves a tweet! 

@bhaveshgohel Would you mind doing the tweet, ccing the contributors? 

I shall re-tweet.

--
Regards,
[Hemanth.HM](https://h3manth.com)

**Comment by hemanth** [comment]
Published  `node-nightly 1.5.0`.

Would you mind passing me your npm handles?

I shall add you as the owner.

**Comment by bhaveshgohel** [comment]
@hemanth Done! [Tweeted](https://twitter.com/ibhaveshgohel/status/762876687096750080) :tada: 

My npm handle is "bhaveshgohel".

**Comment by hemanth** [comment]
```
$ npm owner add bhaveshgohel
+ bhaveshgohel (node-nightly)
```

**Comment by tarungarg546** [comment]
NPM :- [**tarun_19**](https://www.npmjs.com/~tarun_19)

**Comment by hemanth** [comment]
```
$ npm owner add tarun_19
+ tarun_19 (node-nightly)
```

**Comment by kentcdodds** [comment]
Thanks to both of you!

**Pull request merged**